pyface/ui/qt4/workbench/tests/test_workbench_window_layout.py

```diff
@@ -26,11 +26,15 @@ def test_change_of_active_qt_editor(self):
 
         layout = WorkbenchWindowLayout(_qt4_editor_area=mock_split_tab_widget)
 
+        class DummyEvent:
+            def __init__(self, new):
+                self.new = new
+
         # This should not throw
-        layout._qt4_active_editor_changed(None, None)
+        layout._qt4_active_editor_changed(DummyEvent(new=None))
```

Comment:
similar to https://github.com/enthought/pyface/pull/864/files#r562936818 only here the handler was called with arguments.  the handler must take a single event argument, and in the previous case we needed the `new` argument for the old signature to be `None`.  So here I simply created a dummy class to serve the same purpose.  This is a bit of a smell though